searx/engines/openstreetmap.py

```diff
@@ -348,9 +355,13 @@ def get_links(result, user_language):
     """Return links from result['extratags']"""
     links = []
     link_keys = set()
-    for k, mapping_function in VALUE_TO_LINK.items():
-        raw_value = result['extratags'].get(k)
-        if raw_value:
+    extratags = result['extratags']
+    if extratags:
```

Comment:
Just a small comment that's not too important:

Probably nobody agrees with me on that, but I'd prefer doing 
```python
if not extratags:
    return [], set()
```
instead of doing that if clause here.

Reply:
done